R/Rules-methods.R

```diff
@@ -1736,143 +1736,187 @@ setMethod("stopTrial",
             })
 
-######## Stopping Individuals
 
+## --------------------------------------------------
+## Convert stopping rules to character descriptions
+## --------------------------------------------------
+
+##' Convert stopping rules to character descriptions for usage in reporting tables
+##'
+##'
+##' This function outputs a short description of a stopping rule.
+##'
+##' @param x The stopping rule object
+##' \code{\linkS4class{Stopping}}
+##' @param \dots further arguments
+##' @return Stopping rule description
+##'
+##' @export
+##' @keywords methods
+##' @name as.character
+NULL
+
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingMinCohorts",
   definition = function(x, ...) {
-    paste("At least",
+    paste("\u2265",
           x@nCohorts,
-          "cohorts have been observed")
+          "cohorts")
   }
 )
 
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingCohortsNearDose",
   definition = function(x, ...) {
-    paste("At least",
-          x@nCohorts,
-          "cohorts have been observed that lie within",
-          x@percentage,
-          "% of the next best dose")
+      paste("\u2265",x@nCohorts,
+            "cohorts in",
+            x@percentage,
+            "% of NBD")
   }
 )
 
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingPatientsNearDose",
   definition = function(x, ...) {
-    paste("At least",
-          x@nPatients,
-          "patients observed that lie within",
-          x@percentage,
-          "% of the next best dose")
+      paste("\u2265",
+            x@nPatients,
+            "patients in",
+            x@percentage,
+            "% of NBD")
   }
 )
 
+
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingMinPatients",
   definition = function(x, ...) {
-    paste("At least",x@nPatients,"patients dosed")
+      paste("\u2265",
+            x@nPatients,
+            "patients dosed")
   }
 )
 
+
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingTargetProb",
   definition = function(x, ...) {
-    paste("Posterior probability [",
+    paste("[",
           x@target[1],
-          "=< Prob(DLT | dose) <=",
+          "\u2264 prob(DLT | dose) \u2264",
           x@target[2],
-          "] for the next best dose is above",
+          "] for NBD \u2265",
           x@prob)
   }
 )
 
+
+##' @describeIn as.character Convert stopping rules to character description
+##'
 setMethod(
   f = "as.character",
   signature = "StoppingMTDdistribution",
   definition = function(x, ...) {
-    paste("At least",
+    paste("\u2265",
```

Comment:
can you maybe just use the UTF8 character directly here? that would be easier to read